tests/std/tests/P0019R8_atomic_ref/test.cpp

```diff
@@ -133,6 +133,14 @@ void test_int_ops() {
     assert(vy.load() == 0x40);
     assert(rx.load() == 0x33);
     assert(ry.load() == 0x33);
+
+    vx--;
+    rx--;

```

Comment:
We assert that all of these read-modify-write operations change the stored value correctly, but not that they yield the correct result. Should we?

Reply:
yes, added